workflows/cloudify_system_workflows/snapshots/snapshot_restore.py

```diff
@@ -92,6 +93,9 @@ def restore(self):
                 self._restore_credentials(postgres)
                 self._restore_agents()
                 self._restore_deployment_envs(existing_dep_envs)
+                subprocess.check_call([
```

Comment:
We do, but when the restore runs it copies a bunch of files around and messes up the permissions.